src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/OrderIT.java

```diff
@@ -0,0 +1,168 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class OrderIT extends SQLIntegTestCase {
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ORDER);
+    }
+
+    @Test
+    public void simpleOrder() throws IOException {
+        String query = "SELECT id, name FROM elasticsearch-sql_test_index_order ORDER BY id";
+        JSONArray result = getSortExplain(query);
+        Assert.assertThat(1, equalTo(result.length()));
+        JSONObject jsonObject = getSortByField(result, "id");
+        Assert.assertThat("asc", equalTo(getOrderTypeForSortAtIdex(result, 0, "id")));
```

Comment:
consider statically importing assertThat, assertFalse methods

Reply:
Changed from `Assert.someMenthod` to `someMethod`. Realized we don't need to import statically as we are extending from `org.junit.Assert`.